dotnet.sh

```diff
@@ -2,6 +2,12 @@
 
 working_tree_root="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
 
+# Don't resolve runtime, shared framework, or SDK from other locations to ensure build determinism
+export DOTNET_MULTILEVEL_LOOKUP=0
```

Comment:
This matches what the Arcade scripts do here: https://github.com/dotnet/coreclr/blob/master/eng/common/tools.sh#L84